src/sidebarConfig.js

```diff
@@ -103,6 +103,22 @@ const config = {
           label: _('Trial Balance'),
           route: '/report/trial-balance',
         },
+        {
+          label: _('GSTR1'),
+          route: '/report/gstr-1',
+          hidden: async () => {
+            const { country } = await frappe.getSingle('AccountingSettings');
+            return country !== 'India';
+          },
```

Comment:
```javascript
hidden: () => frappe.AccountingSettings.country === 'India',
```

No need to fetch from DB `AccountingSettings` is  cached on the `frappe` object during startup.